release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -81,6 +81,11 @@ For more information, see xref:../installing/install_config/installing-customizi
 
 If a cloud administrator has already set a custom `/etc/chrony.conf` configuration, {op-system} no longer sets the `PEERNTP=no` option by default on cloud platforms. Otherwise, the `PEERNTP=no` option is still set by default. See link:https://bugzilla.redhat.com/show_bug.cgi?id=1924869[BZ#1924869] for more information.
 
+[id="ocp-4-8-rhcos-"multipath-install"]
+==== Enabling multipathing at installation
+
+Enabling multipathing during installation is now supported and recommended for nodes provisioned in {product-title} 4.8 or higher. You can enable multipathing by appending kernel arguments to the `coreos-installer install` command so that the installed system itself will use multipath beginning from the first boot.
```

Comment:
Is there any platform restriction on multipath support? Or do we actually recommend this for all platforms?

Reply:
> Is there any platform restriction on multipath support? > Or do we actually recommend this for all platforms?

Hi @miabbott - My understanding is that it would be recommended for bare metal, so I've clarified that in the wording. Is this correct, @jlebon?

Also, I've clarified the wording so it doesn't sound like _all_ clusters should have multipath enabled. The main point is that we recommend multipathing at install over configuring it as a day-2 operation, not as a general principle.